ui/src/utils/cluster.utils.ts

```diff
@@ -19,10 +19,9 @@ export function generateClusterName(username = ''): string {
   // prepare to combine, but filter any empty part (only one that should ever be empty is user string)
   const nameArray = [userPart, datePart, randomPart].filter(Boolean);
 
-  // combine the 3 parts, truncate it at 28 characters, and remove a trailing '-', if any.
-  // Truncation is needed to allow generating wildcard certificates for OpenShift using Let's Encrypt: since LE insists
-  // on the domain forming the Common Name of the certificate, the string '*.apps.<name>.openshift.infra.rox.systems'
```

Comment:
This justification is confusing given that `openshift-4` and `openshift-4-demo` clusters do not get certs as part of their automation - maybe Malte was adding them post creation for test. (* this will change with #1006)